prediction_market_agent_tooling/markets/seer/price_manager.py

```
@@ -0,0 +1,119 @@
+            self._log_track_price_normalization_diff(
+                old_price=price_yes, normalized_price=normalized_price_yes
+            )
+            return Probability(normalized_price_yes)
+        else:
+            return None
+
+    @lru_cache(typed=True)
+    def get_price_for_token(
+        self,
+        token: ChecksumAddress,
+        collateral_exchange_amount: Wei = xdai_to_wei(xdai_type(1)),
```

Ruff (0.8.2)

Do not perform function call `xdai_to_wei` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)